### PR TITLE
fix void variable consult-preview-key error

### PR DESCRIPTION
**Pull request by LemonBreezes**
This is a proper fix to #108.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -112,9 +112,6 @@ You can also fork this repository and use your own repo.
 **** manual installation
 Clone this repo and make sure the files are on your load path, as described on [[https://www.emacswiki.org/emacs/LoadPath][EmacsWiki]].
 
-*Make sure you load consult (e.g. =require consult=) before you load consult-gh.*
```
Comment (armindarvish) [inline README.org:115]: You should keep this in the README because this is still a good reminder for manual installation!

**Review by armindarvish — CHANGES_REQUESTED** [review]
Fix the README and it should be good to go.

File: README.org (hunk 1 above)
Comment (LemonBreezes) [reply README.org]: Okay. Just removed that change.

**Comment by LemonBreezes** [comment]
> Fix the README and it should be good to go.

Okay. Just removed that change.

**Pull request merged**